Gemfile.lock

```diff
@@ -86,6 +86,7 @@ PATH
       unix-crypt
       warden
       windows_error
+      winrm
```

Comment:
Do you mind reverting the changes to `Gemfile.lock`? They're diverging rapidly with `master`. I can fix it up while landing.

Reply:
Looks like there are still conflicts.

Reply:
Not sure what I did wrong that first time but I tried it again, using the `Gemfile.lock` from the merge base commit be9dda40bd25dae438ba05f5d08fec9730295015.